rest-api-spec/test/msearch/10_basic.yaml

```diff
@@ -33,7 +33,7 @@
           - index: test_2
           - query:
               match_all: {}
-          - search_type: count
+          - search_type: query_then_fetch
```

Comment:
I think the purpose of this test is to check that msearch supports setting the search type?